pyinter/interval.py

```diff
@@ -195,6 +195,14 @@ def _get_new_lower_upper(self, other, operator):
             else:
                 new_upper = self._upper
         return new_lower, new_upper
+    
+    def is_empty(self):
+        """
+        Returns True iff there are no elements inside this interval.
+        """
+        return (self._lower_value >= self._upper_value and
+                self._lower == self.OPEN and
+                self._upper == self.OPEN)
```

Comment:
The interval actually "can't" have `lower_value > upper_value` because the constructor prohibits it. I was just being pedantic but in this case it would probably better not to be.